docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -89,6 +89,22 @@ To modify an existing Pyroscope data source:
 1. Optional: Use **Additional settings** to configure or modify other options.
 1. After completing your updates, select **Save & test**.
 
+#### Connection URL
+
+The data source connection URL should point to a location of a running Pyroscope backend.
+
+**Grafana Cloud Profiles**
+
+A fully configured data source is automatically provisioned in your Grafana Cloud instance.
```

Comment:
Changing the wording slightly to use active instead of passive voice. 
```suggestion
Your Grafana Cloud instance automatically includes a fully provisioned data source.
```